bin/generator.js

```diff
@@ -78,12 +78,24 @@ function bodyGen (obj, kind) {
 function funcObject (obj) {
     var res = esprima.parse(`
         'use strict';
-        var res, indent;
+        var res, indent, spaceNum, spaceString = '';
```

Comment:
@wanderer how about option in generator to insert configurable or hardcoded indent and see on the benchmark?